### PR TITLE
[v2.6] Automate test for RKE2 Authorized Cluster Endpoints

### PR DESCRIPTION
**Pull request by MSpencer87**
Added automated test for RKE2 Authorized Cluster Endpoints

### CODE REVIEW

File: tests/v2/validation/provisioning/rke2/ace_test.go [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+	newUser.Password = user.Password
+
+	standardUserClient, err := client.AsUser(newUser)
+	require.NoError(r.T(), err)
+
+	r.standardUserClient = standardUserClient
+}
+
+func (r *RKE2ACETestSuite) Provisioning_RKE2ClusterACE(provider Provider) {
```
Comment (izaac) [inline tests/v2/validation/provisioning/rke2/ace_test.go:83]: Underscore

File: tests/v2/validation/provisioning/rke2/ace_test.go [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+					}
+					contexts, err := kubeconfig.GetContexts(kubeConfig)
+					require.NoError(r.T(), err)
+					var contextNames []string
+					for key, _ := range contexts {
+						if strings.Contains(key, "pool") {
+							contextNames = append(contextNames, key)
+						}
+					}
+					for _, contextName := range contextNames {
```
Comment (izaac) [inline tests/v2/validation/provisioning/rke2/ace_test.go:190]: What's the negative test here? The "unauthorized" check.

File: tests/v2/validation/provisioning/rke2/ace_test.go (hunk 1 above)
Comment (MSpencer87) [reply tests/v2/validation/provisioning/rke2/ace_test.go]: Removed underscore

**Review by caliskanugur — APPROVED** [review]
Looks good to me

**Pull request closed without merging**